test/functional/fixtures/shell-test.c

```diff
@@ -12,8 +12,12 @@ static void help(void)
   puts("  shell-test");
   puts("  shell-test EXE");
   puts("    Prints \"ready $ \" to stderr.");
+  puts("  shell-test -t {prompt text}");
+  puts("    Prints \"{prompt text} $ \" to stderr.");
   puts("  shell-test EXE \"prog args...\"");
   puts("    Prints \"ready $ prog args...\\n\" to stderr.");
+  puts("  shell-test -t {prompt text} EXE \"prog args...\"");
+  puts("    Prints \"ready {prompt text} $ progs args...\" to stderr.");
```

Comment:
Ah it is used, I just made a typo. It should say:

` Prints \"{prompt text} $ progs args...\" to stderr."`

Will fix now, thanks!